controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
On Wed, Apr 13, 2016 at 01:21:47PM -0700, Nick Anderson wrote:
> I think these should be moved inside the bundle they are used
> in. Set to the default value if there is no def.lldpctl_json
> defined.
> 

Probably a good idea.  Would cover the (e.g. FreeBSD) case where the
path is /usr/local/bin/lldpctl.

-- 
John D. "Trix" Farrar            __\\|//__               Basement.NET
trix@basement.net                (` o-o ')   http://www.basement.net/
--------------------------------ooO-(_)-Ooo--------------------------
 GPG Key Fprint: 525F DBA7 1A62 E4C4 E642  DF95 384B B851 3CEF C10A